mtp_cashbook/templates/faq.html

```diff
@@ -0,0 +1,81 @@
+{% extends 'base.html' %}
+{% load i18n %}
+
+{% block page_title %}{{ view.title }} – {{ block.super }}{% endblock %}
+
+{% block body_classes %}{{ block.super }} mtp-credits{% endblock %}
+
+{% block content %}
+  {{ block.super }}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <a class="govuk-back-link" href="{% url 'home' %}">
+        {% trans 'Back' %}
+      </a>
+
+      <header>
+        <h1 class="govuk-heading-xl">{{ view.title }}</h1>
+      </header>
+
+
+      <h2 class="govuk-heading-m" id="faq-account-requested">
+        {% trans "I requested an account, but heard nothing" %}
+      </h2>
+
+      <p class="govuk-body">
+        {% blocktrans trimmed %}
+          Either remind your business hub manager to accept your request, or <a href="{{ sign_up_url }}" class="govuk-link">request an account</a> again.
+        {% endblocktrans %}
+      </p>
+
+
+      <h2 class="govuk-heading-m" id="faq-locked-out">
+        {% trans "I’m locked out" %}
+      </h2>
+
+      <p class="govuk-body">
+        {% blocktrans trimmed %}
+          If you mis-type your password too many times, you'll be locked out. Try again in about half an hour.
```

Comment:
in general, it's best to split separate sentences into different translation blocks. i know that we'll probably never actually turn on other languages; this would only _theoretically_ make it easier to reuse translated snippets

Reply:
Will do. I thought in this case it was a block which made sense as a whole and that's why it made sense to have in a single block. I can change 👍 

(As a amateur translator it was sometimes tricky to understand the context of certain single sentences)

Reply:
that is indeed true. at the moment, there's no context to _any_ of our translatable strings. it's just one long meaningless table in transifex